frame/staking/src/tests.rs

```diff
@@ -5624,3 +5624,18 @@ fn reducing_max_unlocking_chunks_abrupt() {
 		MaxUnlockingChunks::set(2);
 	})
 }
+
+#[test]
+fn cannot_set_unsupported_validator_count() {
```

Comment:
There are other transaction that also set `ValidatorCount`. They should also be guarded, and tested.

Reply:
`increase_validator_count` and `scale_validator_count`